src/Symfony/Component/String/ByteString.php

```diff
@@ -25,7 +25,7 @@
  */
 class ByteString extends AbstractString
 {
-    private const ALPHABET_ALPHANUMERIC = '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz';
+    private const ALPHABET_ALPHANUMERIC = '0123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz';
```

Comment:
`l`, `I`, `O` are also missing, on purpose, to not use confusable characters

Reply:
Right :)